detoul

```diff
@@ -108,7 +112,7 @@ save-spec() {
   stash
   (
     git checkout detoul-spec &&
-      (cat "$DETOUL_TMP/spec" | head -n -1 | grep -v "^$" > "$target" || [ 1 -eq 1 ]) &&
+      (cat "$DETOUL_TMP/spec" | awk -v n=1 'NR>n{print line[NR%n]};{line[NR%n]=$0}' | grep -v "^$" > "$target" || [ 1 -eq 1 ]) &&
```

Comment:
Please don't add awk as a dependency. Find an alternative solution.

Before it comes up: not python, node, ruby, php, or perl either. We have perl in colorize, but colorize isn't a core file (it can be skipped).

Reply:
The main reason for using awk as a alternative method was OSX's head command doesn't support negative values (so you can't exclude the last line, like you intended here). 
Would you consider coreutils (https://www.gnu.org/software/coreutils/manual/html_node/head-invocation.html) as a polyfil?

Reply:
I understand the problem, please find a solution that doesn't add awk (or another language) as a dependency.

If you can't find a different solution for Mac, you maybe can create create a new executable file here called `skip_last_line` or something that checks the OS, and if it's GNU head (`head --version | head -n1 | grep -q GNU`) uses head and otherwise uses awk.